bosh-setup/azuredeploy.json

```diff
@@ -113,10 +113,10 @@
 
     "baseUriAzureCloud": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/bosh-setup/",
```

Comment:
You mentioned the templates are used across clouds with "standard" tools and scripts. What are the tools? From our understanding, we only need to make sure Azure Portal, CLI and Powershell can deploy the template successfully.
We still think this bp is not applicable to the bosh-setup template because end users never need to change the `baseUriAzureCloud` as a parameter.